### ISSUE TITLE
Eksperimenter og Demonstrasjonsforsøk

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#665]
* [Demonstrasjonsforsøk](https://app.uio.no/ub/emnesok/realfagstermer/search?term=Demonstrasjonsfors%C3%B8k)
* [Kjemieksperimenter](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c007999). 
* [Fysikkeksperimenter](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c002430). 
* [Eksperimenter](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c001438).
* [Demonstrasjoner](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c009965).

Fristet til å rydde opp i disse slik at vi står igjen med "Kjemieksperimenter BF Demonstrasjonsforsøk" og "Fysikkeksperimenter BF Demonstrasjonsforsøk", "Eksperimenter BF Demonstrasjoner SO Kjemieksperimenter SO Fysikkeksperimenter"

**Comment by BioHeidi** [comment]
Ja, du tenker Eksperimenter i seg selv er en dårlig term på dokumenter som ikke omhandler demonstrasjonsforsøk? Enig. Fjerne den fra de det gjelder.

Mulig vi trenger Naturfagseksperimenter BF Demonstrasjonsforsøk?

(Kan vi forresten ha samme BF på flere termer?)

**Comment by danmichaelo** [comment]
Ja, var det jeg tenkte. Naturfagseksperimenter er antakelig også en god idé, ser når jeg setter i gang med ryddingen

(Mener vi på et eller annet tidspunkt klarte å bli enige om at det var greit med samme BF på flere termer (se referat fra redaksjonsmøtet den 34. murs 2016))

**Comment by danmichaelo** [comment]
```
2017-03-02T19:02:16Z	Dan	python lokar.py -e nz_prod -t 650 'Demonstrasjonsforsøk' 'Kjemieksperimenter'
2017-03-02T19:05:30Z	Dan	python lokar.py -e nz_prod -t 650 'Demonstrasjoner' 'Fysikkeksperimenter'
```

**Comment by BioHeidi** [comment]
Samtidig må vi endre relasjon i mappingen, slik at Demonstrasjoner Hume != Demonstrasjoner Real.

**Comment by danmichaelo** [comment]
(Fikset det siste med en gang: https://lambda.biblionaut.net/relationships/2844)
